src/content/en/fundamentals/engage-and-retain/push-notifications/permission-ux.md

```diff
@@ -0,0 +1,177 @@
+demonstrates this approach.
+
+After the user has booked a flight asks if the user would like notifications in case of flight
+delays.
+
+![Owen Campbell-Moore's example of good UX for push.](./images/ux-examples/owen/owen-good-example.png){: .device-image .center-image }
+
+Note that this isn't the browser UI as well, this is a custom UI in the web site asking the
+user, allowing the site full control over the messaging to the user.
+
+Another nice touch to Owen's demo is that if the user does click to enable
+notification, the site adds a semi-transparent overlay over the entire page when
+it shows the permission prompt. This helps draw the users attention to the
+permission prompt.
+
+![Owen Campbell-Moore's example of good UX for the permission prompt.](./images/ux-examples/owen/owen-permission-prompt.png){: .device-image .center-image }
+
+The alternative to this example, the **bad UX** for asking permission, is to  request
+permission as soon as a user lands on the airline's site.
+
+![Owen Campbell-Moore's example of bad UX for push.](./images/ux-examples/owen/owen-bad-ux.png){: .device-image .center-image }
+
+No context as to what the notifications are for, if the user wanted to get a
+task done (i.e. check a flight or book a flight), this permission prompt gets in
+the way of that and secondly it's not a good look having a pop-up over the site.
+
+### Double Permission
+
+You may feel that your site has a clear need for push messaging and as a result
+want to ask the user for permission as soon as possible. An example of this
+sites that fit into this category are messaging sites or email clients. You
+receiving a message or email, show a notification.
+
+In these cases, it's worth considering the double permission UX.
+
+With this approach you display a custom permission prompt in your web app which
+asks the user to enable notifications. By doing this the user can say enable or
+disable without risk of being permanently blocked. If the user selects enable on
+the custom UI, display the actual permission prompt, otherwise hide your custom
+pop-up and ask some other time.
+
+A good example of this is [Slack](https://slack.com/). They show a prompt at
+the top of their page once you've signed in asking if you'd like to enable notifications.
+
+
+
+### Settings Panel
+
+You can move notifications into a settings panel, giving users an easy way
+to enable and disable push messaging, without the need of cluttering your
+web app's UI.
+
+A good example of this is [Google I/O's 2016 site](https://events.google.com/io2016/). When you
+first load up the Google I/O site, you aren't asked to do anything,
+the user is left to explore the site.
+
+![When you first load the page, no prompt, just calm on Google IO.](./images/ux-examples/google-io/google-io-first-load.png){: .device-image .center-image }
+
+After a few visits of clicking the menu item on the right, a settings panel is reveal to the
```

Comment:
So they can't even see the menu item until they've made visits that specifically involve clicking the menu? If that's not what you meant, please take another shot at this sentence.